categories.py

```diff
@@ -245,6 +246,9 @@
    "RecoLocalCalo/HcalRecProducers",
    "RecoLocalCalo/HGCalRecAlgos",
    "RecoLocalCalo/HGCalRecProducers",
+   "RecoLocalFastTime/Configuration",
+   "RecoLocalFastTime/FTLCommonAlgos",
+   "RecoLocalFastTime/FTLRecProducers",
```

Comment:
Do these have to have "Common" and "Rec" in the name?
What else do we expect in the RecoLocal for FTL?